Xamarin.Essentials/HapticFeedback/HapticFeedback.android.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Android.Views;
+
+namespace Xamarin.Essentials
+{
+    public static partial class HapticFeedback
+    {
+        internal static bool IsSupported => true;
+
+        static async Task PlatformExecute(HapticFeedbackType type)
+        {
+            try
+            {
+                Platform.CurrentActivity?.Window?.DecorView?.PerformHapticFeedback(ConvertType(type));
+                await Task.FromResult(0);
```

Comment:
Maybe use `Task.CompletedTask`. And no need to async. Just return it directly.

Reply:
I added everywhere "await Task.FromResult(0)" only because "await"is used on UWP. Tell me how best to remove it.